src/main/java/edu/ucsb/cs156/happiercows/controllers/CommonsController.java

```diff
@@ -124,6 +126,23 @@ public Commons getCommonsById(
     return ResponseEntity.ok().body(body);
   }
 
+  @ApiOperation(value = "Delete a Commons")
+  @PreAuthorize("hasRole('ROLE_ADMIN')")
+  @DeleteMapping("")
+  public Object deleteCommons(
+          @ApiParam("id") @RequestParam Long id) {
+      
+      Commons foundCommons = commonsRepository.findById(id).orElseThrow( ()->new EntityNotFoundException(Commons.class, id));
+ 
+      commonsRepository.deleteById(id);
+      userCommonsRepository.deleteAllByCommonsId(id);
```

Comment:
This destroys both the `COMMONS` and `USER_COMMONS` tables.

Reply:
no way to fix this with the given setup it seems.